libphal/utils_pdbg.H

```diff
@@ -42,5 +42,14 @@ struct pdbg_target *getFsiTarget(struct pdbg_target *proc);
  **/
 struct pdbg_target *getTgtFromBinPath(const ATTR_PHYS_BIN_PATH_Type &binPath);
 
+/**
+ *  @brief  Helper function to check target is a processor type
+ *
+ *  @param[in]  tgt - target to check
+ *
+ *  Exceptions: PdbgError
+ */
+void isProcTgt(struct pdbg_target *tgt);
```

Comment:
should we call the function name as validateIfProcTarget as we are validating

Reply:
ack